src/common/manual-entry-file.ts

```diff
@@ -0,0 +1,4 @@
+export interface DictionaryEntry {
```

Comment:
I like the new interface, but could you please put it in in `src/common/types.ts`? I don't want there to be an explosion of files. Not every interface needs its own source file!

Also, please add JSDoc comments!